### PR TITLE
Fixes #223: heroku deployment link added to mail

### LINKED ISSUE #223: send heroku deployment link in mail
### This is a
- [ ] Bug report
- [x] Feature request

### Current behavior

now we sending only GitHub deploy link
### Expected behavior

send Heroku deployment link as well in the mail

### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

heroku deployment url added to email template
## Related Issue

close #223 
## Motivation and Context

it'll allow the users go to heroku deploy url directly from email
## Screenshots (if appropriate):
redered email http://big-hole.surge.sh/
renderedemail
## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: backend/mailer.js [hunk 1]
```diff
@@ -20,19 +20,24 @@ exports.sendEmail = function (data) {
...
-    deployURL = 'http://' + hostname + '/github?email=' + data.email + '&uniqueId=' + data.uniqueId + '&gitURL=' + data.gitUrl;
+    githubDeployURL = 'http://' + hostname + '/auth/github?email=' + data.email + '&uniqueId=' + data.uniqueId + '&gitURL=' + data.gitUrl;
   }
+  var herokuDeployURL = 'http://' + hostname + '/auth/heroku?email='+data.email+'&uniqueId='+data.uniqueId;
 
   var textContent = 'Hey! Your documentation generated successfully. Preview it here: ' + previewURL +
                     '. Download it here: ' + downloadURL ;
   if (deployURL !== "") {
-    textContent += '. Deploy it here: ' + deployURL;
+    textContent += '. GitHub Deploy: ' + githubDeployURL;
```
Comment (pri22296) [inline backend/mailer.js]: Github Deploy -> Deploy to Github Pages

File: backend/mailer.js [hunk 2]
```diff
@@ -20,19 +20,24 @@ exports.sendEmail = function (data) {
...
   }
+
+  textContent += '. Heroku Deploy: ' + herokuDeployURL;
```
Comment (pri22296) [inline backend/mailer.js]: Heroku Deploy -> Deploy to Heroku

File: backend/mailer.js (hunk 1 above)
Comment (sch00lb0y) [inline backend/mailer.js]: made changes

File: backend/mailer.js (hunk 2 above)
Comment (sch00lb0y) [inline backend/mailer.js]: made changes

**Pull request merged**